### DISCUSSION TITLE
Raspberry pi alternative?

### DISCUSSION THREAD

**Discussion by jhollis505** [awawa-dev/HyperHDR#454] [Q&A]
Is there any alternative soc for the raspberry pi? I'm having a hard time finding a rpi for a decent price.

**Comment by awawa-dev** [comment]
Hi
As you can see on my blog, there are many tvboxes that work with HyperHDR even without code recompilation using aarch64 version. The key is how stable is Armbian OS release for the device. You may experience problems when you plan to use SPI (you must check if Armbian supports it for the board) or use USB3.0 port for a grabber (some platforms USB3.0 kernel drivers are not as stable as others and there must be a platform specific config present to make it work). And forget about using ws281x directly from the pinout...still HyperSerialEsp8266/HyperSerialESP32 should work.
